Yoast/wordpress-seo
# Add days to how to block structured data

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Also removed `title`, which was removed from the How To block  in #10889

## Test instructions

This PR can be tested by following these steps:

* In gutenberg, create a how-to block with a duration, including at least 1 day.
* Press `preview`. Inspect the output of the How To block in the post.
* The output of the how to blick is prepended with a `div` with type `application/ld+json`.
* Copy the contents of this div into `https://search.google.com/structured-data/testing-tool/u/0/?hl=nl`
* Verify that the duration format in the structured data is correct, using https://en.wikipedia.org/wiki/ISO_8601#Durations.

## Quality assurance


Fixes #10892
